guides/doc-Content_Management_Guide/topics/Managing_Errata.adoc

```diff
@@ -310,20 +307,18 @@ Use these procedures to review and apply errata to a host.
 * Synchronize {ProjectName} repositories with the latest errata available from Red{nbsp}Hat.
 For more information, see xref:Importing_Content-Synchronizing_Repositories[].
 
-* Register the host to an environment and Content View on {ProjectServer}.
+* Register the host to an environment and Content View on the {ProjectServer}.
```

Comment:
This is fine for Foreman but a super taboo for Satellite. I have no idea why. It makes things awkward but we've generally gone for article-less Foreman server.